elvish-mode.el

```diff
@@ -63,7 +63,7 @@ expected.")
 
 (defconst elvish-keyword-pattern
   (let ((keywords (cons 'or elvish-keywords)))
-    (eval `(rx (group ,keywords) (not word))))
+    (eval `(rx (not (any word ?_ ?-)) (group ,keywords) (not (any word ?_ ?-)))))
```

Comment:
Also, after this patch, a file starting with `use ...` does not have the `use` highlighted.